packages/ndla-ui/src/Grid/Grid.tsx

```diff
@@ -6,102 +6,81 @@
  *
  */
 
-import { HTMLAttributes, ReactNode } from "react";
-import styled from "@emotion/styled";
-import { breakpoints, colors, misc, mq, spacing } from "@ndla/core";
+import { ReactNode } from "react";
+import { styled } from "@ndla/styled-system/jsx";
+
+const GridContainer = styled("div", {
+  base: {
+    display: "grid",
+    justifyContent: "center",
+    borderRadius: "xsmall",
+    padding: "xxlarge",
+    gridRowGap: "large",
+    gridColumnGap: "medium",
+    width: "100%",
+    marginBottom: "medium",
+    backgroundColor: "background.subtle",
+    maxWidth: "surface.4xlarge",
+    minWidth: "surface.xxsmall",
+    gridTemplateColumns: "repeat(2, minmax(0, 1fr))",
+    tabletDown: {
+      gridTemplateColumns: "repeat(1, minmax(0, 1fr))",
+      "& > div:nth-child(3):last-child": {
+        gridColumn: "span 1",
+      },
+    },
+    desktopDown: {
+      gridTemplateColumns: "repeat(2, minmax(0, 1fr))",
+      "& > div:nth-child(3):last-child": {
+        display: "flex",
+        flexFlow: "column",
+        justifyContent: "center",
+        alignItems: "center",
+        gridColumn: "span 2",
+      },
+    },
+  },
+  variants: {
+    columns: {
+      "2": {},
+      "2x2": {},
+      "3": { tabletWide: { gridTemplateColumns: "repeat(3, minmax(0, 1fr))" } },
+      "4": { tabletWide: { gridTemplateColumns: "repeat(4, minmax(0, 1fr))" } },
```

Comment:
Syns dere dette er måten å gjøre dette på?

Reply:
Synes det er helt fint! Men trenger vi ikke noe spesialhåndtering for 2x2?

Reply:
Hvordan da tenker du på?

Egentlig så kunne nok `2` og `2x2` vært slått sammen til samme prop verdi, men jeg tenkte å ikke gjøre den breaking.
Kommer jo bare an på hva du sender inn som children om den wrapper eller ikke.

Vil du jeg skal fjerne `2x2` muligheten og bare ha `2`?